src/FileConventions.Test/DummyFiles/DummyWithCRLFLineEndings

```diff
@@ -0,0 +1,3 @@
+line1
+line2
+line3

```

Comment:
@realmarv this is confusing; what's the difference between this file and DummyWithoutMixedLineEndings file? because it seems but have line endings which are not mixed

Reply:
@realmarv ^?

Reply:
I renamed DummyWithoutMixedLineEndings file. Is that ok?

Reply:
that sounds better, but why did you remove DummyWithCRLFLineEndings?

Reply:
DummyWithCRLFLineEndings is added in the third commit